### ISSUE TITLE
Bug in MIDI Preferences Panel

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#2366]
Nigel Rios writes:

> **What went wrong?**
> When trying to select an action in the MIDI Control Surface Panel, it is not selected. I tried to select AudioSwift 3 device first, but it resets after choosing the action.
> 
> **What are the steps to recreate the problem?**
> Launched FCP, launched CP, went to Control Surface, and select an action.

```lua
CommandPost v1.0.7-beta.10 (Build: 6035)

Build Date: � � � � � � � � � � Apr 29 2020, 01:02:27
macOS Version:� � � � � � � � � 10.15.4
CommandPost Locale: � � � � � � en
Current Keyboard Layout:� � � � Spanish - ISO
Final Cut Pro Path: � � � � � � /Applications/Final Cut Pro Trial.app
Final Cut Pro Version:� � � � � 10.4.8
Final Cut Pro Language: � � � � en
Developer Mode: � � � � � � � � false

2020-06-21 21:24:56: 2020-06-21 21:24:54: -- Loading /Applications/CommandPost.app/Contents/Resources/extensions/cp/init.lua
2020-06-21 21:24:56: 21:24:56 � � � � cp: � � Loading Plugins...
2020-06-21 21:25:00: ********
2020-06-21 21:25:00: 21:25:00 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:728: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:881: in method 'update'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:1060: in field 'postInit'
	...st.app/Contents/Resources/extensions/cp/plugins/init.lua:635: in function 'cp.plugins.postInitPlugin'
	...st.app/Contents/Resources/extensions/cp/plugins/init.lua:580: in function 'cp.plugins.postInitPlugins'
	...st.app/Contents/Resources/extensions/cp/plugins/init.lua:688: in function 'cp.plugins.init'
	...ommandPost.app/Contents/Resources/extensions/cp/init.lua:323: in function <...ommandPost.app/Contents/Resources/extensions/cp/init.lua:53>
	(...tail calls...)
	[C]: in function 'xpcall'
	...app/Contents/Resources/extensions/hs/_coresetup/init.lua:794: in function 'hs._coresetup.setup'
	(...tail calls...)
2020-06-21 21:25:00: ********
2020-06-21 21:25:00: ********
2020-06-21 21:25:00: 21:25:00 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:881: in method 'update'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:1061: in field 'postInit'
	...st.app/Contents/Resources/extensions/cp/plugins/init.lua:635: in function 'cp.plugins.postInitPlugin'
	...st.app/Contents/Resources/extensions/cp/plugins/init.lua:580: in function 'cp.plugins.postInitPlugins'
	...st.app/Contents/Resources/extensions/cp/plugins/init.lua:688: in function 'cp.plugins.init'
	...ommandPost.app/Contents/Resources/extensions/cp/init.lua:323: in function <...ommandPost.app/Contents/Resources/extensions/cp/init.lua:53>
	(...tail calls...)
	[C]: in function 'xpcall'
	...app/Contents/Resources/extensions/hs/_coresetup/init.lua:794: in function 'hs._coresetup.setup'
	(...tail calls...)
2020-06-21 21:25:00: ********
2020-06-21 21:25:00:� � � � � � � � � cp: � � Plugins Loaded.
2020-06-21 21:25:00:� � � � � � � � � � � � � Startup Time: 3.875793 seconds
2020-06-21 21:25:00: -- Done.
2020-06-21 21:25:51: ********
2020-06-21 21:25:51: 21:25:51 ERROR: � LuaSkin: hs.webview.usercontent callback: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:354: in upvalue 'updateUI'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:894: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:666>
	(...tail calls...)
2020-06-21 21:25:51: ********
2020-06-21 21:25:56: ********
2020-06-21 21:25:56: 21:25:56 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:99: in upvalue 'setItem'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:834: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:666>
	(...tail calls...)
2020-06-21 21:25:56: ********
2020-06-21 21:26:02: ********
2020-06-21 21:26:02: 21:26:02 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:99: in upvalue 'setItem'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:834: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:666>
	(...tail calls...)
2020-06-21 21:26:02: ********
2020-06-21 21:26:07: ********
2020-06-21 21:26:07: 21:26:07 ERROR: � LuaSkin: hs.webview.usercontent callback: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:354: in upvalue 'updateUI'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:429: in upvalue 'stopLearning'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:899: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:666>
	(...tail calls...)
2020-06-21 21:26:07: ********
2020-06-21 21:26:13: ********
2020-06-21 21:26:13: 21:26:13 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:99: in upvalue 'setItem'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:834: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:666>
	(...tail calls...)
2020-06-21 21:26:13: ********
2020-06-21 21:26:25: ********
2020-06-21 21:26:25: 21:26:25 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:766: in field '_onActivate'
	...ents/Resources/plugins/core/action/manager/activator.lua:1165: in method 'activate'
	...ents/Resources/plugins/core/action/manager/activator.lua:856: in function <...ents/Resources/plugins/core/action/manager/activator.lua:851>
2020-06-21 21:26:25: ********
2020-06-21 21:26:25: ********
2020-06-21 21:26:25: 21:26:25 ERROR: � LuaSkin: hs.chooser:completionCallback: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:354: in upvalue 'updateUI'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:769: in field '_onActivate'
	...ents/Resources/plugins/core/action/manager/activator.lua:1165: in method 'activate'
	...ents/Resources/plugins/core/action/manager/activator.lua:856: in function <...ents/Resources/plugins/core/action/manager/activator.lua:851>
2020-06-21 21:26:25: ********
2020-06-21 21:26:47: ********
2020-06-21 21:26:47: 21:26:47 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:99: in upvalue 'setItem'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:834: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:666>
	(...tail calls...)
2020-06-21 21:26:47: ********
2020-06-21 21:26:53: ********
2020-06-21 21:26:53: 21:26:53 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:766: in field '_onActivate'
	...ents/Resources/plugins/core/action/manager/activator.lua:1165: in method 'activate'
	...ents/Resources/plugins/core/action/manager/activator.lua:856: in function <...ents/Resources/plugins/core/action/manager/activator.lua:851>
2020-06-21 21:26:53: ********
2020-06-21 21:26:53: ********
2020-06-21 21:26:53: 21:26:53 ERROR: � LuaSkin: hs.chooser:completionCallback: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:354: in upvalue 'updateUI'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:769: in field '_onActivate'
	...ents/Resources/plugins/core/action/manager/activator.lua:1165: in method 'activate'
	...ents/Resources/plugins/core/action/manager/activator.lua:856: in function <...ents/Resources/plugins/core/action/manager/activator.lua:851>
2020-06-21 21:26:53: ********
2020-06-21 21:28:13: ********
2020-06-21 21:28:13: 21:28:13 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:99: in upvalue 'setItem'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:834: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:666>
	(...tail calls...)
2020-06-21 21:28:13: ********
2020-06-21 21:28:21: ********
2020-06-21 21:28:21: 21:28:21 ERROR: � LuaSkin: hs.timer callback error: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:354: in upvalue 'updateUI'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:1339: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:1336>
2020-06-21 21:28:21: ********
2020-06-21 21:28:22: ********
2020-06-21 21:28:22: 21:28:22 ERROR: � LuaSkin: hs.timer callback error: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:354: in upvalue 'updateUI'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:1339: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:1336>
2020-06-21 21:28:22: ********
2020-06-21 21:28:30: ********
2020-06-21 21:28:30: 21:28:30 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:766: in field '_onActivate'
	...ents/Resources/plugins/core/action/manager/activator.lua:1165: in method 'activate'
	...ents/Resources/plugins/core/action/manager/activator.lua:856: in function <...ents/Resources/plugins/core/action/manager/activator.lua:851>
2020-06-21 21:28:30: ********
2020-06-21 21:28:30: ********
2020-06-21 21:28:30: 21:28:30 ERROR: � LuaSkin: hs.chooser:completionCallback: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:354: in upvalue 'updateUI'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:769: in field '_onActivate'
	...ents/Resources/plugins/core/action/manager/activator.lua:1165: in method 'activate'
	...ents/Resources/plugins/core/action/manager/activator.lua:856: in function <...ents/Resources/plugins/core/action/manager/activator.lua:851>
2020-06-21 21:28:30: ********
2020-06-21 21:28:34: ********
2020-06-21 21:28:34: 21:28:34 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:99: in upvalue 'setItem'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:834: in function <....app/Contents/Resources/plugins/core/midi/prefs/init.lua:666>
	(...tail calls...)
2020-06-21 21:28:34: ********
2020-06-21 21:28:44: ********
2020-06-21 21:28:44: 21:28:44 ERROR:� � � prop: Error while notifying a watcher: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: attempt to index a number value (local 'button')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:124: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:540: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:525>
	(...tail calls...)
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:766: in field '_onActivate'
	...ents/Resources/plugins/core/action/manager/activator.lua:1165: in method 'activate'
	...ents/Resources/plugins/core/action/manager/activator.lua:856: in function <...ents/Resources/plugins/core/action/manager/activator.lua:851>
2020-06-21 21:28:44: ********
2020-06-21 21:28:44: ********
2020-06-21 21:28:44: 21:28:44 ERROR: � LuaSkin: hs.chooser:completionCallback: ...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: attempt to index a number value (local 'item')
stack traceback:
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:634: in field 'start'
	...pp/Contents/Resources/plugins/core/midi/manager/init.lua:714: in field 'update'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:354: in upvalue 'updateUI'
	....app/Contents/Resources/plugins/core/midi/prefs/init.lua:769: in field '_onActivate'
	...ents/Resources/plugins/core/action/manager/activator.lua:1165: in method 'activate'
	...ents/Resources/plugins/core/action/manager/activator.lua:856: in function <...ents/Resources/plugins/core/action/manager/activator.lua:851>
2020-06-21 21:28:44: ********
```

**Comment by latenitefilms** [comment]
Nigel writes:

> 1. Is there an option to turn off the “Switching to MIDI bank” message? I was thinking on staying with one view in AudioSwift and change between banks with CP to control the different color wheels. But some people may not like the message constantly appearing.
> 
> 2. How does the Scrub Timeline (Relative) works? AudioSwift has a Jog Wheel and I tried to map this action, but it doesn’t move. This jog wheel sends values of 1 clockwise, and value of 65 counterclockwise. I could do it with Go To Previous/Next Frame, but AudioSwift loses its focus and the mouse pointers moves; it’s not what I’m looking for.
> 
> 3. Does CP have an action to reset the last parameter used? As I think I mentioned before, AudioSwift works with Lightroom via MIDI2LR, and there is an action like this to reset the parameter.

**Comment by latenitefilms** [comment]
To be addressed in #2410.
